File: src/main/java/com/lee/Ticket.java
package com.lee;

public class Ticket {
    int start;
    int end;
    int price;

    public Ticket() {
    }

    public Ticket(int start, int end, int price) {
        this.start = start;
        this.end = end;
        this.price = price;
    }

    public int getStart() {
        return start;
    }

    public void setStart(int start) {
        this.start = start;
    }

    public int getEnd() {
        return end;
    }

    public void setEnd(int end) {
        this.end = end;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }
}

File: src/main/kotlin/com/lee/Ticket.kt
package com.lee.kotlin

import kotlin.random.Random

data class Ticket(val start :Int , val end :Int , val price:Int)

fun main(args: Array<String>) {

    var ticket =Ticket(20,30,100)

    var string ="abcdefg"
    println(string.validateLength())
    var string2 ="ab"
    println(string2.validateLength())

    println((1..10).random())
}
fun String.validateLength() : Boolean{
    return this.length >=6
}
fun IntRange.random():Int{
    return Random.nextInt(endInclusive-start)+start
}
